internal/providers/util/backoff.go

```diff
@@ -1,30 +0,0 @@
-// Copyright 2015 CoreOS, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package util
-
-import (
-	"time"
-)
-
-// ExpBackoff calculates an exponential (power 2) backoff given the last
-// backoff duration and the maximum backoff duration.
-func ExpBackoff(backoff *time.Duration, maxBackoff time.Duration) time.Duration {
```

Comment:
Let's link to 198184c720b08b3945c498765faf4349757bd64f in the commit message.

Reply:
Done.